package.json

```diff
@@ -37,7 +37,7 @@
     "lint:docs-configs": "node tests/validate-docs.js",
     "lint:css": "cd docs && stylelint css/*.css",
     "docs": "cd docs && bundle exec jekyll serve",
-    "postinstall": "opencollective postinstall || (exit 0)",
+    "postinstall": "opencollective-postinstall",
```

Comment:
The ` || (exit 0)` fixed the postinstall script sometimes causing the install to fail. Could you please readd it?

Reply:
Done.